addons/mail/static/src/models/thread_view_topbar.js

```diff
@@ -556,6 +559,12 @@ registerModel({
          * `doSetSelectionEndOnThreadDescriptionInput` to have an effect.
          */
         doSetSelectionStartOnThreadDescriptionInput: attr(),
+        fileUploader: one('FileUploader', {
+            default: {},
+            inverse: 'threadViewTopbarOwner',
+            isCausal: true,
+            readonly: true,
+        }),
```

Comment:
I feel a file uploader in the thread view topbar lack a bit context by itself.
It's not super clear it's related to avatar of the channel.

Could rename field to `avatarFileUploader`, so it's clearer!